### PR TITLE
Improve logs while draining

### PR DESCRIPTION
**Pull request by hardikdr**
**What this PR does / why we need it**: This PR improves the log while draining - adds node-name wherever needed.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: pkg/controller/drain.go [hunk 1]
```diff
@@ -603,21 +603,21 @@ func (o *DrainOptions) evictPodsWithPVInternal(attemptEvict bool, pods []*corev1
...
 		} else if err != nil {
-			glog.V(4).Infof("Error when evicting pod: %v/%v. Will be retried. Err: %v", pod.Namespace, pod.Name, err)
+			glog.V(4).Infof("Error when evicting pod: %v/%v from node %v. Will be retried. Err: %v", pod.Namespace, pod.Name, pod.Spec.NodeName, err)
 			retryPods = append(retryPods, pod)
 			continue
 		}
 
 		// Eviction was successful. Wait for pvs for this pod to detach
-		glog.V(3).Info("Eviction/delete succeeded. Waiting for the volumes to detach: ", pod.Name)
+		glog.V(3).Info("Eviction/delete succeeded. Waiting for the volumes to detach: ", pod.Name, pod.Spec.NodeName)
```
Comment (prashanth26) [inline pkg/controller/drain.go]: We need to capture the time taken for eviction/deletion of the pod here. We might have to introduce a time variable for the same.

File: pkg/controller/drain.go [hunk 2]
```diff
@@ -632,7 +632,7 @@ func (o *DrainOptions) evictPodsWithPVInternal(attemptEvict bool, pods []*corev1
 			returnCh <- err
 			continue
 		}
-		glog.V(3).Info("Detached pv for pod: ", pod.Name)
+		glog.V(3).Info("Detached pv for pod: ", pod.Name, " from node ", pod.Spec.NodeName)
```
Comment (prashanth26) [inline pkg/controller/drain.go]: We need to capture the time taken for detachment of PV here. We might have to introduce a time variable for the same or reuse the one used for PV eviction logic above.

File: pkg/controller/drain.go [hunk 3]
```diff
@@ -785,14 +785,14 @@ func (o *DrainOptions) evictPodWithoutPVInternal(attemptEvict bool, pod *corev1.
 		if err == nil {
 			break
 		} else if apierrors.IsNotFound(err) {
-			glog.V(3).Info("\t", pod.Name, " evicted")
+			glog.V(3).Info("\t", pod.Name, " evicted from node ", pod.Spec.NodeName)
```
Comment (prashanth26) [inline pkg/controller/drain.go]: We need to capture the time taken for eviction/deletion of the pod here. We might have to introduce a time variable for the same.

**Comment by hardikdr** [comment]
Agreed, we need more improvements in general for logging including the one mentioned above. I suggest, let's take it in subsequent PRs, and scope this one with current quick change.

**Comment by prashanth26** [comment]
Okay. Shall accept these changes but keep the issue - #305 open to track the same.

**Pull request merged**